bfcfg

```diff
@@ -1758,6 +1759,52 @@ sb_cfg()
   fi
 }
 
+#
+# Dump configurations stored in EFI variable needed by OS applications
+#
+os_cfg_dump()
+{
+
+  local tmp_file=${TMP_DIR}/.bfcfg-osfs-data
+
+  if [ ${dump_level} -eq ${DUMP_LEVEL_NONE} ]; then
+    return
+  fi
+
+  if [ ! -e "${os_cfg_sysfs}" ]; then
+    log_msg "os-cfg: failed to find the ${os_cfg_sysfs} EFI variable"
+    return
+  fi
+
+  # shellcheck disable=SC2216
+  yes | cp -f ${os_cfg_sysfs} ${tmp_file}
+  enableLLDPextensions=$(hexdump -s 4 -n 1 -e '/1 "%d" "\n"' ${tmp_file})
+
+  # shellcheck disable=SC2216
+  enableDMS=$(hexdump -s 5 -n 1 -e '/1 "%d" "\n"' ${tmp_file})
+
+  echo "os: ENABLE_LLDP_EXTENSIONS=${enableLLDPextensions}"
```

Comment:
I think know I understand why you would expect the variable to be created by default. So I would leave it up to you then. If the intention is to have these config options printed all the time, then yes, I would agree to have the EFI variable present all the time.

Reply:
Ack